testing/e2e/skr-tester/pkg/broker/client.go

```diff
@@ -246,25 +246,15 @@ func (c *BrokerClient) ProvisionInstance(instanceID, planID, region string, btpO
 	return c.CallBroker(payload, endpoint, "PUT")
 }
 
-func (c *BrokerClient) UpdateInstance(instanceID string, customParams, btpOperatorCreds map[string]interface{}, isMigration bool) (map[string]interface{}, error) {
+func (c *BrokerClient) UpdateInstance(instanceID string, customParams map[string]interface{}) (map[string]interface{}, error) {
 	payload := map[string]interface{}{
 		"service_id": kymaServiceID,
 		"context": map[string]interface{}{
 			"globalaccount_id": c.GlobalAccountID,
-			"isMigration":      isMigration,
 		},
 		"parameters": customParams,
 	}
 
-	if btpOperatorCreds != nil {
```

Comment:
Is update of SAP BTP service operator credentials no longer supported?

Reply:
Not at the moment. I will add it when needed.

Reply:
I want to keep it clean and add functionality that is needed. Previously it was added more as a placeholder as the function was rewritten from nodejs.

Reply:
Ok, let's focus on machine type update in this PR. I think we will need the possibility to update the credentials in the context for one of the e2e tests, but we can add it later.